### DISCUSSION TITLE
Is Detekt support Code Coverage with sonarqube ?

### DISCUSSION THREAD

**Discussion by dineshoptisol** [detekt/detekt#5535] [Q&A] [answered]
Hi, i recently using **Detekt + sonarqube** for static analysis and i enabled the Detekt inputs path for test files also. 
Luckily i got report/codesmells in **SonarQube** but my Coverage showing always 0.0%. 

is anyway to show under coverage? or is Detekt support code coverage with **sonarqube** ?

Thanks

**Comment by schalkms** [comment, marked as answer]
Detekt is a static code analyzer and hence does not support code coverage. Other tools exist exactly for this purpose. Based on the given information, I doubt that, just because you enabled detekt, code coverage is not reported. Did you try a run **without** detekt being part of your project?

**Reply by dineshoptisol** [reply]
Hi, Yeah I’m moved to jacoco for code coverage. Just got a doubt whether Detekt is supporting codecoverage or not. If it supports i no need to use jacoco right so that's way asked.

But as you said and some reference i got from web shows “its not support code coverage”. Now i got clarity. thanks man!!
